doc/translation_strings_policy.md

```diff
@@ -82,7 +82,7 @@ This adds `<numerusform>`s to the respective `.ts` file, which can be translated
         </translation>
     </message>
 
-Where it is possible try to avoid embedding numbers into the flow of the string at all. e.g.
+Where possible, try to avoid embedding numbers into the flow of the string at all. e.g.
```

Comment:
Same here.
Also the first letter should capitalized after a full stop, right?

Reply:
good catch, will update!